### ISSUE TITLE
Unexpected logs 

### ISSUE THREAD

**Issue by lapter57** [xnuinside/simple-ddl-parser#78]
Hi, with the new version 0.19.8 I get unexpected logs when calling `DDLParser(...).run(...)`. For example:
```
WARNING: Token 'NEWLINE' defined, but not used
WARNING: There is 1 unused token
CREATE CREATE
TABLE TABLE
IF IF
NOT NOT
EXISTS EXISTS
shema ID
table ID
field_1 ID
BIGINT ID
, COMMA
`partition` ID
STRING ID
) RP
```

**Comment by xnuinside** [comment]
Ah, sorry for this, will clean it up today in 0.19.9 and will add a test for log output

**Comment by xnuinside** [comment]
@lapter57  released 0.19.9 with issues fix https://pypi.org/project/simple-ddl-parser/. Thanks for the report!
